jung-visualization/src/main/java/edu/uci/ics/jung/visualization/PluggableRenderContext.java

```diff
@@ -105,11 +104,10 @@
   protected GraphicsDecorator graphicsContext;
 
   private EdgeShape<E> edgeShape;
-
+  //Function<Context<Network,E>, Shape>
   PluggableRenderContext(Network<V, E> graph) {
-    this.network = graph;
-    this.edgeShape = new EdgeShape<E>(graph);
-    this.edgeShapeTransformer = edgeShape.new QuadCurve();
+    //    this.edgeShape = new EdgeShape<E>(graph);
```

Comment:
Ditto.

Please look back over all the other comments you added and consider whether they're really needed.